spyderlib/spyder.py

```diff
@@ -2255,10 +2254,12 @@ def maximize_dockwidget(self, restore=False):
 
     def __update_fullscreen_action(self):
         if self.isFullScreen():
-            icon = "window_nofullscreen.png"
+            icon = ima.icon('collapse')

```

Comment:
In that case the two icons are essentially the same. don't you think we could collapse it to a single entry?